AdyenNetworking/APIClient/APIClient.swift

```diff
@@ -174,6 +179,9 @@ public final class APIClient: APIClientProtocol {
         _ request: R
     ) throws -> HTTPResponse<R.ResponseType> {
         log(result: result, request: request)
+        
+        try responseValidator?.validateReponse(result.data, for: request, with: result.headers)
```

Comment:
would be nice to unit test this? assert that the validator is called using a mock validator

Reply:
updated, thanks